tide_core.module

```diff
@@ -189,3 +190,19 @@ function tide_core_entity_bundle_create($entity_type_id, $bundle) {
     }
   }
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function tide_core_preprocess_status_messages(&$variables) {
+  if (isset($variables['message_list']['error']) && !empty($variables['message_list']['error'])) {
```

Comment:
We probably want to get more specific with the context of when this message is altered i.e. only when it relates to `Related links`

Reply:
Hi @GROwen , I added more specific checkers, which will make sure it only happens under nod context.

Reply:
@GROwen I think the new condition that I added will be enough for checking the context.